index.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Salmon Cookies</title>
+  <meta name="description" content="The HTML5 Herald">
+  <meta name="author" content="SitePoint">
+
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+<!-- /*<style>
+body {
+    background-image: url("img/salmon.png" alt="picture of salmon during spawning season");
+}
+</style>*/ -->
+</head>
+
+<body>
+  <img src="img/salmon.png" alt="picture of salmon during spawning season" width=100%>
+
+  <h1>Salmon Cookies</h1>
+  <p>New intovation in coffee cookie dipping delight! Coming to a shop near you soon...</p>
+    <h2>More Coming Soon</h2>
+
+ <!-- <script src="js/scripts.js"></script> -->
```

Comment:
Since this is your public facing page, does it need this script? If not, I would remove this to remove the commented out code.